compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -341,6 +338,8 @@ def __init__(
           'reward_key not specified in ModuleExplorer initialization.')
     self._reward_key = reward_key
     kwargs.pop('reward_key', None)
+    gin_config_str = kwargs.pop('gin_config_str', None)
+    gin.parse_config(gin_config_str)
```

Comment:
add a `gin.clear_config()` here. In test, for example, previous tests that do a `gin.bind_parameter` can mess things up.

separate issue: what we need to do with `gin.bind_parameter` in tests (in a maintainable way)